src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CompilerExtensions.projitems

```diff
@@ -486,4 +487,7 @@
   <ItemGroup Condition="'$(DefaultLanguageSourceExtension)' != '' AND '$(BuildingInsideVisualStudio)' != 'true'">
     <ExpectedCompile Include="$(MSBuildThisFileDirectory)**\*$(DefaultLanguageSourceExtension)" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
What does this do?

Reply:
basically, it gives us a folder in the UI where this linked file goes.  it seems needed because we have our own file called SpecialTypeExtensions, and we want to link in yours.